github-custom-fonts.user.css

```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         GitHub Custom Fonts
 @namespace    StylishThemes
-@version      4.2.130
+@version      4.2.144
```

Comment:
dont bump versions pls

Reply:
This will cause conflicts at merge.

Reply:
This Bumpup was when rebased from Master.
I think to merge Master and version together.

Reply:
IDK how you rebased, but a proper rebase wouldn't introduce changes that you haven't added manually.

lets assume you setup an upstream (named) remote when you forked this repo. if not replace `upstream` with the name or your remote.

run `git pull --rebase upstream master` then followed by `git push --force`

That should be round about it.